examples/on-conflict-clause/on_conflict_clause.bal

```diff
@@ -1,38 +1,61 @@
 import ballerina/io;
 
-type Student record {|
+type Employee record {|
     readonly int id;
     string name;
+    string role;
     int score;
```

Comment:
Why score instead of something like salary?

Reply:
+1 for salary